resources/js/components/felte-form/FelteForm.tsx

```diff
@@ -79,14 +86,14 @@ export const FelteForm = <T extends Obj = Obj>(props: FormProps<T>) => {
         });
       }
     },
-  });
+  }) as FormType<T>;
 
   const TypedFormContext = typedFormContext<T>();
   return (
     <TypedFormContext.Provider value={{props, form: form as FormType<T>, translations}}>
       <form ref={form.form} {...formProps}>
         <fieldset class="contents" disabled={form.isSubmitting()} inert={form.isSubmitting() || undefined}>
-          {local.children}
+          {getChildrenElement(local.children, form)}
```

Comment:
Hmm dałoby radę Dynamic tu użyć? Bez niego trochę się boję o nadmiar rerenderów

Reply:
a samej funkcji getChildrenElement wewnątrz przydałby się <Show />

Reply:
Rerendery nie powinny się zdarzyć żadne. No bo jeśli children jest funkcją to nigdy się nie zmieni (chyba że ktoś odwali coś naprawdę dziwnego, no ale to jest prawda uniwersalna w JS). Natomiast jeżeli jest JSX to nie wiem czy się zmieni czy nie, ale to bez znaczenia, bo i tak `getChildrenElement(local.children, form)` jest po prostu równoważne z napisaniem `local.children`.

Co do wnętrza funkcji, Show nie jest potrzebny, no chyba że ktoś pojedzie już tak po bandzie, że children zmieni się z funkcji w JSX albo odwrotnie - spokojnie zakładam że nie.